lib/tmuxinator/window.rb

```diff
@@ -34,13 +34,16 @@ def initialize(window_yaml, index, project)
 
     def build_panes(panes_yml)
       Array(panes_yml).map.with_index do |pane_yml, index|
-        if pane_yml.is_a?(Hash)
-          pane_yml.map do |_name, commands|
-            Tmuxinator::Pane.new(index, project, self, *commands)
-          end
-        else
-          Tmuxinator::Pane.new(index, project, self, pane_yml)
-        end
+        commands =  case pane_yml
+                    when Hash
+                      pane_yml.values.first
```

Comment:
Yeah I wrestled with the right behavior here.  If you could check out #406 and give me your thoughts.

If you think about the yaml structure, its not clear what exactly multiple hash elements would represent.  I gave a few options in that issue above.